src/scene/materials/standard-material.js

```diff
@@ -1076,6 +1086,7 @@ function _defineMaterialProps() {
     _defineTex2D('diffuse', 0, 3, '', true);
     _defineTex2D('specular', 0, 3, '', true);
     _defineTex2D('emissive', 0, 3, '', true);
+    _defineTex2D('specularityFactor', 0, 1, '', false);
```

Comment:
Looks like `specularityFactorPS` supports vertex color:
```suggestion
    _defineTex2D('specularityFactor', 0, 1, '', true);
```
Is there no default, expected channel to find specularityFactor? (like alpha maybe?)